tools/rimage/config/mtl.toml

```diff
@@ -601,9 +601,10 @@ count = 27
 	[[module.entry]]
         name = "RTC_AEC"
         uuid = "B780A0A6-269F-466F-B477-23DFA05AF758"
-        affinity_mask = "0x3"
+        # bit #i = 1 means core #i is allowed.
```

Comment:
I'd prefer a "rimage: mtl: " prefix for this commits.

Reply:
Ack, will update in next version.